bugbug/repository.py

```diff
@@ -206,22 +219,81 @@ def download_commits(repo_dir, date_from):
         if cut is not None:
             commits_by_author[commit.author] = commits_by_author[commit.author][cut + 1:]
 
-        author_experience_90_days[commit] = len(commits_by_author[commit.author])
+        author_experience_90_days[commit.node] = len(commits_by_author[commit.author])
 
         if not commit.backedoutby:
             commits_by_author[commit.author].append(commit)
 
+    print()
+    print('Time (author): {}'.format(time.perf_counter() - start))
+
     global COMPONENTS
     r = requests.get('https://index.taskcluster.net/v1/task/gecko.v2.mozilla-central.latest.source.source-bugzilla-info/artifacts/public/components.json')
     r.raise_for_status()
     COMPONENTS = r.json()
+    COMPONENTS = {fl: '::'.join(cmp) for fl, cmp in COMPONENTS.items()}
```

Comment:
Nit: rename `cmp` to `component`, as `cmp` is a reserved keyword.

Use `path` instead of `fl`, here and elsewhere.